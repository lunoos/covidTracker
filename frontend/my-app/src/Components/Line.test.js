import { shallow } from 'enzyme';
import LineGraph from './Bar';
import react from 'react';

it('expect to render LineGraph component', () => {
	
	expect(shallow(<LineGraph />)).toMatchSnapshot();
})