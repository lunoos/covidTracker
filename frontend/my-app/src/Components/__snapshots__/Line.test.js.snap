// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render LineGraph component 1`] = `ShallowWrapper {}`;
